### ISSUE TITLE
Minifying causes Leaflet not to load

### ISSUE THREAD

**Issue by JKetelaar** [myparcelnl/magento#407]
**Describe the bug**
Whenever "Minify JavaScript Files" is turned on, Magento adds .min to the external Leaflet script.

**To Reproduce**
1. Enable Minify JavaScript Files
2. Go to checkout
3. See error in console

```
require.min.js:120 GET https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.min.js net::ERR_ABORTED 404
...
Uncaught Error: Script error for: leaflet
http://requirejs.org/docs/errors.html#scripterror
    at makeError (require.min.js:16)
    at HTMLScriptElement.onScriptError (require.min.js:112)
```

**Expected behavior**
Loading https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.js instead of https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.min.js

**Screenshots**
![image](https://user-images.githubusercontent.com/3681904/81698630-8eb4a580-9466-11ea-9644-0201014ecd21.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - MyParcel Plugin version:
```
            "name": "myparcelnl/magento",
            "version": "dev-new-delivery-options",
```
 - MyParcel SDK version: 
```
            "name": "myparcelnl/sdk",
            "version": "v3.1.7",
```

**Additional context**
Related to magento/magento2#5835

**Possible solution**
Adding the following code to config.xml
```!xml
        <dev>
            <js>
                <minify_exclude>
                    <leaflet>https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.js</leaflet>
                </minify_exclude>
            </js>
        </dev>
```

**Comment by RichardPerdaan** [comment]
#415

**Comment by RichardPerdaan** [comment]
This issue is resolved in version [4.0.0-beta.1] (https://github.com/myparceln/magento/releases/tag/4.0.0-beta.1)
